db/seeds.rb

```diff
@@ -6,25 +6,32 @@
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
+Category.destroy_all
+EventTemplate.destroy_all
+
 template_list = [
-    ['Green', 'http://res.cloudinary.com/neddinn/image/upload/v1438782351/temp13_ovz0ue.png '],
-    ['Blue', 'http://res.cloudinary.com/neddinn/image/upload/v1438782351/temp15_hj7rqz.png'],
-    ['Purple', 'http://res.cloudinary.com/neddinn/image/upload/v1438782350/temp7_doqim8.png '],
-    ['Red','http://res.cloudinary.com/neddinn/image/upload/v1438782351/temp12_mbqjcp.png '],
-    ['Black', 'http://res.cloudinary.com/neddinn/image/upload/v1438782350/temp8_y8qt9v.png'],
-    ['Gray', 'http://res.cloudinary.com/neddinn/image/upload/v1438782350/temp6_ng6krr.png'],
-    ['Sky-blue','http://res.cloudinary.com/neddinn/image/upload/v1438782350/temp2_dy7nkp.png '],
-    ['Brown', 'http://res.cloudinary.com/neddinn/image/upload/v1438782351/temp9_lhchbz.png']
+  ['blue','http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402582/blue_k0b4yt.png'],
+  ['teal','http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402583/teal_yajwze.png'],
+  ['red','http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402583/red_ug5h6q.png'],
+  ['orange','http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402583/orange_btitzu.png'],
+  ['lime','http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402583/lime_xlfi2l.png' ],
+  ['light-blue','http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402583/light-blue_o5ildf.png'],
+  ['grey', 'http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402582/grey_bfdrfw.png'],
+  ['pink', 'http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402583/pink_uzoho9.png'],
+  ['green','http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402582/green_r7ixce.png'],
+  ['cyan','http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402582/cyan_r5ptzy.png'],
+  ['brown','http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402582/brown_jzwgq6.png'],
+  ['purple','http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402583/purple_rryrhz.png']
 ]
 
 category_list = [
-    ['Music', 'Intimate, house concerts, major music festival, and the occasional dance party', ''],
-    ['Food & drink', 'Dinner parties, tasting and big-time festivals', ''],
-    ['Classes', 'Enlightening seminars, technical workshops, and fitness classes', ''],
-    ['Arts', 'Plays, comedy nights, art exhibitions and film festivals', ''],
-    ['Parties', 'Casual happy hours, singles nights, and all-night celebrations', ''],
-    ['Sport & Wellness', 'Obstacle races, drop-in yoga classes, and the big game', ''],
-    ['Networking', 'Business mixers, hobby meetups, and panel discussions', '']
+  ['Music','Intimate, house concerts, major music festival, and the occasional dance party',''],
```

Comment:
Line is too long. [96/80]<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.